recipes-mapbox/qmapboxgl/qmapboxgl/config.gypi

```diff
@@ -0,0 +1,37 @@
+{
+  'target_defaults': {
+    'cflags%': [],
+    'default_configuration': 'Release',
+    'defines': [],
+    'include_dirs': [],
+    'libraries': [],
+    'cflags': ['-fvisibility=hidden'],
+  },
+  'variables': {
+    'qt_version_major%': ['5'],
+    'boost_cflags%': [],
+    'coverage%': [],
+    'geojsonvt_cflags%': [],
+    'geojsonvt_ldflags%': [],
+    'geometry_cflags%': [],
+    'geometry_ldflags%': [],
+    'geometry_static_libs%': [],
+    'gtest_cflags%': [],
+    'gtest_ldflags%': [],
+    'gtest_static_libs%': [],
+    'nunicode_cflags%': [],
+    'nunicode_ldflags%': [],
+    'pixelmatch_cflags%': [],
+    'pixelmatch_ldflags%': [],
+    'pixelmatch_static_libs%': [],
+    'protozero_cflags%': [],
+    'rapidjson_cflags%': [],
+    'rapidjson_ldflags%': [],
+    'rapidjson_static_libs%': [],
+    'sqlite_static_libs%': [],
+    'variant_cflags%': [],
+    'variant_ldflags%': [],
+    'variant_static_libs%': [],
+    'zlib_static_libs%': [],
+  }
+}
```

Comment:
These are the empty variables. If you don't define it, mbgl build system will complain. I left it in a separated file so it doesn't pollute the recipe.